contracts/scripts/_contractsLocalSetup.js

```diff
@@ -91,6 +91,7 @@ const outputJsonConfigFile = async (outputPath) => {
     let outputDictionary = {}
     outputDictionary['registryAddress'] = registry.address
     outputDictionary['ownerWallet'] = await getDefaultAccount()
+    outputDictionary['allWallets'] = await web3.eth.getAccounts()
```

Comment:
is this actually used anywhere in this updated flow? it seems to me like an index is passed and the pkey info recoveered through the contract files

Reply:
nvm i see it used here but idk if it even needs to? https://github.com/AudiusProject/audius-tooling/pull/24/files